registry/scheduledTasks.xml

```diff
@@ -59,4 +59,8 @@
 		<descr>Process the usage stats logs and load the numbers into the DB tables.</descr>
 		<frequency day="0"/>
 	</task>
+	<task class="lib.pkp.classes.task.ProcessQueueJobs">
+		<descr>Process pending queue jobs in the default queue driver and queue</descr>
+		<frequency minute="0"/>
```

Comment:
We discussed changing this to `second="0"` to get it to run every minute, but at a glance it looks like `lib/pkp/classes/scheduledTask/ScheduledTaskHelper.php` will need to be extended to include that option.

Reply:
Done extending .